tensorflow/compiler/xla/service/cpu/BUILD

```diff
@@ -350,7 +351,7 @@ cc_library(
         "windows_compatibility.h",
     ],
     hdrs = ["simple_orc_jit.h"],
-    copts = tf_copts(),
+    copts = if_enable_acl(["-DXLA_CPU_USE_ACL=1", "-fexceptions"]),
```

Comment:
@snadampal `XLA_CPU_USE_ACL` doesn't appear in any of the source files. I don't think we need either copts here?

Reply:
@penpornk , this is required for the header files included in these targets. for example, 
https://github.com/tensorflow/tensorflow/blob/master/tensorflow/compiler/xla/service/cpu/simple_orc_jit.cc#L47
includes
#include "tensorflow/compiler/xla/service/cpu/runtime_matmul_acl.h

and samilarly, dot_op_emitter includes cpu_runtime that has the definitions required for acl.

Reply:
Maybe then we can move XLA_CPU_USE_ACL closer to usage?

Reply:
@snadampal Thank you for the clarifications! I think `-DXLA_CPU_USE_ACL=1` is needed in `:simple_orc_jit` then. Could you please try without `-fexceptions`? Nothing in those `*acl.h` header files should require exceptions.

> and samilarly, dot_op_emitter includes cpu_runtime that has the definitions required for acl.

I'll continue in a different thread on `:dot_op_emitter`.

> Maybe then we can move XLA_CPU_USE_ACL closer to usage?

@cheshire Can you clarify what you mean by moving `XLA_CPU_USE_ACL` closer to the usage?